stories/Tab.stories.ts

```diff
@@ -0,0 +1,71 @@
+import { Story, Meta } from '@storybook/angular/types-6-0';
+import { TabComponent } from 'projects/ion/src/lib/tab/tab.component';
+
+export default {
+  title: 'Ion/Tab',
+  component: TabComponent,
+} as Meta;
+
+const Template: Story<TabComponent> = (args: TabComponent) => ({
+  component: TabComponent,
+  props: args,
+});
+
+export const Basic = Template.bind({});
+Basic.args = {
+  label: 'Custom label',
+};
+
+export const TabSmall = Template.bind({});
+TabSmall.args = {
+  label: 'Small',
+  size: 'sm',
```

Comment:
```suggestion
  tabSize: 'sm',
```
Your args should have the same name that your component's inputs.

Reply:
Same for the args below.